posts/2024-02-12-Java21-Images.adoc

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "Liberty container images with Semeru Runtime Java 21 support"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/tjwatson
+author_github: https://github.com/tjwatson
+seo-title: Liberty container images with Semeru Runtime Java 21 support
+seo-description: Liberty now offers container images with Java 21 support using the Semeru Runtime JRE. These images use UBI 9 minimal base images and the Semeru JRE to produce smaller application images. They also now offer InstantOn support for the Power and Z (s390x) processor architectures, along with the previously supported x86-64/AMD64 processor architectures. 
+blog_description: "Liberty now offers container images with Java 21 support using the Semeru Runtime JRE. These images use UBI 9 minimal base images and the Semeru JRE to produce smaller application images. They also now offer InstantOn support for the Power and Z (s390x) processor architectures, along with the previously supported x86-64/AMD64 processor architectures."
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= Liberty container images with Semeru Runtime Java 21 support
+Thomas Watson <https://github.com/tjwatson>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+Liberty now offers container images with Java 21 support using the Semeru Runtime JRE. These images use UBI 9 minimal base images and the Semeru JRE to produce smaller application images. They also now offer InstantOn support for the Power and Z (s390x) processor architectures, along with the previously supported x86-64/AMD64 processor architectures.
+
+== Universal Base Image Minimal Version 9
+The link:https://catalog.redhat.com/software/containers/ubi9/ubi-minimal/615bd9b4075b022acc111bf5[Universal Base Image 9 Minimal] is a stripped down image that uses `microdnf` as a package manager, which allows for the production of smaller application images. Open Liberty Java 21 UBI 9 minimal link:https://github.com/OpenLiberty/ci.docker/blob/main/docs/icr-images.md[images] are available starting with the 24.0.0.1 release of Open Liberty. The UBI 9 minimal content may impact some application image builds because of missing functionality. Application image builds can use `microdnf` to install additional packages as required. At the time of this publication UBI 9 is not FIPS compliant. Other Open Liberty Java version images will continue to use the UBI 8 standard images (Java 8, Java 11, Java 17).
```

Comment:
```suggestion
The link:https://catalog.redhat.com/software/containers/ubi9/ubi-minimal/615bd9b4075b022acc111bf5[Universal Base Image 9 Minimal] is a stripped down image that uses `microdnf` as a package manager, which allows for the production of smaller application images. Open Liberty Java 21 UBI 9 minimal link:https://github.com/OpenLiberty/ci.docker/blob/main/docs/icr-images.md[images] are available starting with the 24.0.0.1 release of Open Liberty. The UBI 9 minimal content may impact some application image builds because of missing functionality. Application image builds can use `microdnf` to install additional packages as required. Unlike UBI 8, UBI 9, at the time of this publication, is not FIPS compliant. Other Open Liberty Java version images will continue to use the UBI 8 standard images (Java 8, Java 11, Java 17).
```